app/services/flow/flows/citizen_dependants.rb

```diff
@@ -18,6 +18,13 @@ class CitizenDependants < FlowSteps
         },
         other_dependant: {
           path: '[PLACEHOLDER] do you have any other dependant?'
+        },
+        dependants_full_time_educations: {
+          # path: ->(_) { urls.citizens_dependant_full_time_education_path },
+          forward: ->(_) { :dependants_incomes }
```

Comment:
you need to pass the `dependant` here

Reply:
Why? No matter how the question is answered, the next page is aways 'dependants_incomes'. The flow is only affected by the answer on leaving 'dependant_incomes'

Reply:
Ah yes sorry you're right.

Reply:
I had to look at it a couple of times to get my head around the flow diagramme

Reply:
I just remembered that you can just write ` forward: :dependants_incomes` here